alertnow/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "cdb258cc-5e74-4fa2-be21-1489375bb370",
+  "app_id": "alertnow",
+  "classifier_tags": [
```

Comment:
Do you mind moving this `classifier_tags` section under the `tile` section?

Reply:
OK, I will edit the file according to your comments.